driver/random.hpp

```diff
@@ -2,15 +2,25 @@
 #define GUARD_RANDOM_GEN_
 
 #include <cstdlib>
+#include <ctime>
+#include <cstdint>
+#include <chrono>
 
 template <typename T>
 inline T FRAND()
 {
-    double d = static_cast<double>(rand() / (static_cast<double>(RAND_MAX)));
+    auto const clk    = std::chrono::system_clock::now();
+    unsigned int seed = static_cast<unsigned int>(std::chrono::system_clock::to_time_t(clk));
```

Comment:
`to_time_t` converts nanoseconds to seconds (
```
auto const clk = std::chrono::system_clock::now().time_since_epoch().count();
unsigned int seed = static_cast<unsigned int>(clk);
```

Reply:
Thanks for that! I was looking for it rn.